idseq_pipeline/commands/non_host_alignment.py

```diff
@@ -6,9 +6,10 @@ class Non_host_alignment(Base):
 
     def run(self):
         from .non_host_alignment_functions import *
-
         unbuffer_stdout()
-        lazy_run = not big_version_change_from_last_run(self.version, get_alignment_version_s3_path())
         upload_commit_sha(self.version)
+        stage1_version = os.environ.get('HOST_FILTER_PIPELINE_VERSION')
```

Comment:
The version is actually available in a variable in the running pipeline.  It should not be obtained from the environment like this.

Care should be taken in the docker container init for pipeline stages >1 to checkout a git hash that is exactly the same as the hash that ran in stage 1.

Reply:
it's used for comparison purpose. this command is for alignment. it wouldnt really know how host filtering was run unless informed